translate/storage/fpo.py

```diff
@@ -32,6 +32,7 @@
 import re
 import six
 from io import BytesIO
+import logging
```

Comment:
Please place this comment in the proper place. Check our styleguide http://docs.translatehouse.org/projects/translate-toolkit/en/latest/developers/styleguide.html#imports